R/get_timescale_data.R

```diff
@@ -0,0 +1,107 @@
+#' Get geological timescale data
+#'
+#' This function takes a name of a geological timescale and returns data for the
+#' timescale. Valid names include those of built-in `data.frames` ([GTS2020()]
```

Comment:
```suggestion
#' timescale. Valid names include those of built-in `data.frame`(s) ([GTS2020()]
```
Maybe? I guess we normally put backticks just around classes/functions/etc?